README.md

```diff
@@ -1,7 +1,9 @@
 # SHT4X
 
-[![Hex version](https://img.shields.io/hexpm/v/sht4x.svg 'Hex version')](https://hex.pm/packages/sht4x)
-[![CircleCI](https://dl.circleci.com/status-badge/img/gh/elixir-sensors/sht4x/tree/main.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/elixir-sensors/sht4x/tree/main)
+[![Hex version](https://img.shields.io/hexpm/v/sht4x.svg "Hex version")](https://hex.pm/packages/sht4x)
+[![API docs](https://img.shields.io/hexpm/v/sht4x.svg?label=hexdocs "API docs")](https://hexdocs.pm/sht4x/SHT4X.html)
+[![CircleCI](https://circleci.com/gh/nerves-project/sht4x.svg?style=svg)](https://circleci.com/gh/nerves-project/sht4x)
+[![REUSE status](https://api.reuse.software/badge/github.com/nerves-project/sht4x)](https://api.reuse.software/info/github.com/nerves-project/sht4x)
```

Comment:
Could you update your `nerves_licenses` repo? There was an issue with generating the tags and I fixed it. You should see a change here next time you run the badge update script. (nerves-project -> elixir-sensors)